### DISCUSSION TITLE
DataBricks

### DISCUSSION THREAD

**Discussion by jtgooding42** [TabularEditor/TabularEditor3#1024] [Ideas]
Is there a way to get Azure Databricks connector working when using Enterprise/Consultancy edition with Power BI Premium/PPU as the workspace and deploy target.  I get Databricks is unknown type error when trying to import a table with a valid Databricks M query.  I would think it would/could be setup similar to SnowFlake as an implicit source.

Thanks
John

**Comment by otykier** [comment]
Native databricks support is on the roadmap. For now, you'll have to use the [workaround described here](https://docs.tabulareditor.com/te3/tutorials/importing-tables.html#updating-table-schema-through-analysis-services).

**Comment by mlonsk** [comment]
Native databricks support exist in the newer versions of Tabular Editor 3.
